target/scripts/startup/setup.d/dovecot.sh

```diff
@@ -87,8 +87,8 @@ function _setup_dovecot_sieve
   fi
 
   chown dovecot:root -R /usr/lib/dovecot/sieve-*
-  find /usr/lib/dovecot/sieve-* -type d -exec chmod 755 {} \;
-  chmod +x /usr/lib/dovecot/sieve-{filter,pipe}/*
+  find /usr/lib/dovecot/sieve-*             -type d -exec chmod 755 {} +
```

Comment:
Curious why we set `chmod 755` and not `chmod -R 755` since this is only matching directories? If we're fixing permissions there, I assume that might be relevant to the contents too :thinking:

Reply:
The fix appears to be correct to me, since we do not want to adjust the contents of the dirs, only the dirs, so the `-R` is not needed and not desired as well (as contents and dirs need different dirs).

If you were using `-R`, you'd be adjusting the contents as well, even though the `find` only matches dirs.